README.md

```diff
@@ -4,64 +4,53 @@
 [![Build Status](https://img.shields.io/travis/ice-lab/ice-scripts.svg?style=flat)](https://travis-ci.org/ice-lab/ice-scripts)
 [![NPM downloads](https://img.shields.io/npm/dm/ice-scripts.svg?style=flat)](https://npmjs.org/package/ice-scripts)
 
-> ice-scripts 是 React 项目的工程构建工具，配置简单、插件化能力，参考文档 [ice-scripts](https://ice.work/docs/cli/about)
+> 🐒Configurable build tool for React project based on webpack. Find detailed instructions in [its documentation](https://ice.work/docs/cli/about)
 
-## 快速上手
+## Features
 
-### 初始化项目
+`ice-scripts` have everything you need to build a React app:
 
-> 也可通过 iceworks GUI 工具进行初始化
+* fully configurable via `ice.config.js`, allow your project to have it's configration
+* the plugin system provides rich feature and allow the community to build reusable solutions
+* out of the box support for ES6+, TypeScripts, Less, Sass, CSS Modules
+* easy to modify built-in wepack configuration by webpack-chain
+* delightful javaScript testing based on Jest
 
-安装 `iceworks` 依赖：
+## Getting Started
+
+> Recommend to create a React app via [iceworks](https://ice.work/iceworks)
 
 ```bash
+# Install deps
 $ npm install iceworks -g
-$ iceworks --help
-```
-
-创建一个空目录：
 
-```bash
+# create an empty folder
 $ mkdir iceapp && cd iceapp
-```
-
-初始化项目：
 
-```bash
+# create react project
 $ iceworks init
-# 或者根据指定模板创建项目
-$ iceworks init @icedesign/pro-scaffold
-
-# 向项目里添加区块
-$ cd src/components/
-$ iceworks add @icedesign/user-landing-block
 ```
 
-完成项目初始化后既可以开始开始项目调试开发和项目构建。
-
-### 调试开发
-
-项目目录下启动调试服务：
+Once the initialization is done, inside the created project, you can run some built-in commands:
 
 ```bash
 $ npm start
 ```
 
-开始调试服务后，可以访问 `http://localhost:4444` 进行页面预览。修改源码内容后将自动刷新页面。
-
-### 构建代码
+Runs the app in development mode.
 
-构建项目代码：
+It will open `http://localhost:4444` for preview. The page will automatically reload if you make change to the code.
```

Comment:
> The page will automatically reload if you make change to the code.

minor detail: The page will be automatically reloaded if you make changes to the code.